instrumentation/spring/spring-boot-resources/library/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/resources/SpringBootServiceNameGuesser.java

```diff
@@ -0,0 +1,274 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.spring.autoconfigure.resources;
```

Comment:
I wonder if we should name the package `io.opentelemetry.instrumentation.spring.resources` and skip the `autoconfigure` part -- so that it doesn't get confused with the `spring-autoconfigure` module

Reply:
I'm all about shorter package names, so I'm into it. :)